features/0509-action-menu/README.md

```diff
@@ -0,0 +1,211 @@
+# Aries RFC 0509: Action Menu Protocol 0.9
+
+- Authors: [Andrew Whitehead],[Stephen Curran]
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-12-26 (date you submit your PR)
```

Comment:
Please review any template instructions, and update the dates in the header.

Reply:
Done